Unaccountable-Magpie/BangazonSite
# remove migration, add new migrations, change SQLServer in Startup file

### Purpose:

To fix the models and migrations in order to allow the DB to build properly.

### How it fits in context of the project:

Fixes problems with running migrations and prepares us for future tickets' needs (like "city").

### Specific Feature affected:

Data models for Product, PaymentType, and ApplicationUser

### How to test (Be thorough!):

1. `git fetch -a`
1. `git checkout el-fixModels`
1. `start Bangazon.sln`
1. Open the (NuGet) Package Manager Console.
1. run the command `update-database` to run the migrations included in this PR.
1. See if there are any errors in either running this migration or build errors that prevent the migration running at all.
1. If the migration runs without errors, open your SQL Server Object Explorer.
1. Open `Databases > BangazonSite > Tables > dbo.AspNetUsers`; right click on this table and choose `View Data` See if there is a User.
1. Next choose `View Data` for the Payment Types. See if there are two.
